crates/ruma-common/src/events/room/message/reply.rs

```diff
@@ -97,17 +95,20 @@ fn formatted_or_plain_body(
 /// [HTML tags and attributes]: https://spec.matrix.org/v1.4/client-server-api/#mroommessage-msgtypes
 /// [rich reply fallbacks]: https://spec.matrix.org/v1.4/client-server-api/#fallbacks-for-rich-replies
 pub fn plain_and_formatted_reply_body(
-    body: impl fmt::Display,
-    formatted: Option<impl fmt::Display>,
+    formatted: Option<&FormattedBody>,
+    body: &str,
     original_message: &OriginalRoomMessageEvent,
 ) -> (String, String) {
     let (quoted, quoted_html) = get_message_quote_fallbacks(original_message);
 
     let plain = format!("{quoted}\n{body}");
-    let html = match formatted {
-        Some(formatted) => format!("{quoted_html}{formatted}"),
-        None => format!("{quoted_html}{body}"),
-    };
+
+    #[cfg(feature = "unstable-sanitize")]
+    let body_html = formatted_or_plain_body(formatted, body, true);
```

Comment:
Not quite sure what `is_reply` should be set to here; it would affect whether or not the formatted body of the reply (produced by the application) is sanitized.

Reply:
It should probably have been set to `false`, but with my others comments this call would go away.